packages/maskbook/src/plugins/ITO/types.ts

```diff
@@ -19,10 +19,13 @@ export interface JSON_PayloadInMask {
     chain_id: ChainId
     start_time: number
     end_time: number
+    unlock_time: number
```

Comment:
IMO new properties should be optional. Otherwise program will have wrong assumption when dealing with old payload.

Reply:
All old ITO were outdated.